src/openqaoa-core/pyproject.toml

```diff
@@ -1,3 +1,60 @@
 [build-system]
 requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+where = ["."]
+
+[project]
+name = "openqaoa-core"
+version = "0.2.5"
+authors = [{ name = "Entropica Labs" }]
+description = "OpenQAOA is a python open-source multi-backend Software Development Kit to create, customise and execute the Quantum Approximate Optimisation Algorithm (QAOA) on Noisy Intermediate-Scale Quantum (NISQ) devices, and simulators"
+readme = "README.md"
+requires-python = ">=3.9, <3.12"
+license = { file = "LICENSE" }
+urls = { "Homepage" = "https://github.com/entropicalabs/openqaoa" }
+classifiers = [
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Operating System :: OS Independent"
+]
+keywords = ["quantum", "optimisation", "SDK"]
+
+dependencies = [
+    "pandas>=2.2.2",
+    "sympy>=1.12",
+    "numpy>=1.23",
+    "networkx>=2.8",
+    "matplotlib>=3.6, <3.9",
+    "scipy>=1.9.0",
+    "docplex==2.25.236",
+    "autograd>=1.6",
+    "semantic_version>=2.10",
+    "autoray>=0.6",
+    "requests>=2.32.3",
+    "ipython>=8.18.1",
+    "mitiq>=0.30.0, <0.36.0"
```

Comment:
This should be corrected by #310 , try to pull the changes from dev and see if you still need to limit the version of `mitiq`

Reply:
Yes! Thanks